github/packages/github/github_find_pr.js

```diff
@@ -36,16 +36,8 @@ function formatDate(d) {
 }
 
 async function _command(params, commandText, secrets = {}) {
-    
-  if (!secrets.github_token) {
-    return {

```

Comment:
This was checked but not used?!

Reply:
it was not being used/passed in headers, but doing so would lead to greater rate limits, will make that optional.